dts/bindings/gpio/seeed-xiao-header.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 Foundries.io
+# Copyright (C) 2019 Peter Bigot Consulting, LLC
+# Copyright (C) 2021 Peter Johanson
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    GPIO pins exposed on Seeesuino Xiao (and compatible devices) headers.
```

Comment:
```suggestion
    GPIO pins exposed on Seeeduino Xiao (and compatible devices) headers.
```

Reply:
This is what I get for working on PRs while still learning colemak-dhm... Fixed.